src/main/java/fi/jubic/quanta/models/ExternalClient.java

```diff
@@ -21,6 +21,11 @@
 
     public abstract String getToken();
 
+    @Nullable

```

Comment:
Is nullable required here? Isn't an empty string enough to indicate that there's no description?

Reply:
Removed @Nullable